gcloud/bigtable/row.py

```diff
@@ -257,6 +257,78 @@ def to_pb(self):
         return data_pb2.RowFilter(value_regex_filter=self.regex)
 
 
+class ValueRangeFilter(RowFilter):
+    """A range of values to restrict to in a row filter.
+
+    Will only match cells that have values in this range.
+
+    Both the start and end value can be included or excluded in the range.
+    By default, we include them both, but this can be changed with optional
+    flags.
+
+    :type start_value: bytes
+    :param start_value: The start of the range of values. If no value is
+                        used, it is interpreted as the empty string
+                        (inclusive) by the backend.
```

Comment:
Hmm, "empty string (inclusive)" above.  Would it be better to declare the effect, rather than an imagined implementation?  E.g., "If no value is passed, the backend applies no lower bound to the values."